django_loci/widgets.py

```diff
@@ -46,7 +46,5 @@ class LeafletWidget(BaseLeafletWidget):
     geom_type = 'GEOMETRY'
     template_name = 'leaflet/admin/widget.html'
     modifiable = True
-    map_width = '100%'
-    map_height = '400px'
```

Comment:
Adding context:

This was deprecated in Django 4.2. [django-leaflet 0.30.0 handled this change](https://github.com/makinacorpus/django-leaflet/pull/363).